contribs/gmf/src/directives/search.js

```diff
@@ -965,7 +965,7 @@ gmf.SearchController.prototype.fulltextsearch_ = function(query, resultIndex, op
         const feature = format.readFeature(data.features[resultIndex - 1]);
         this.featureOverlay_.addFeature(feature);
         const fitOptions = /** @type {olx.view.FitOptions} */ ({});
-        if (opt_zoom) {
+        if (opt_zoom !== null) {
```

Comment:
if `opt_zoom` is not passed to the function, it will be `undefined` not `null`

Reply:
And the `@param` for `opt_zoom` should be changed to:
```
@param {number=} opt_zoom Optional zoom level.
```

See https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#optional-parameter-in-a-param-annotation